### DISCUSSION TITLE
How to manually manage author photos

### DISCUSSION THREAD

**Discussion by Gajotres** [advplyr/audiobookshelf#1319] [Q&A] [answered]
What is the new way of manually updating author photos?

In the previous versions, I could always manually add a photo to the **metadata/authors** folder and have this photo shown in the Audiobookshelf UI. In the current 2.2.11 version, no matter what I do, Audiobookshelf is always showing the originally acquired author's photo.

I've also noticed that **metadata/authors** folder is no longer used for author photos. For example, even though the UI shows the author photo URL as **/metadata/authors/aut_0z33fid50f4wx0o7us.jpg**, this image is not there. I'm also unable to find this particular file (aut_0z33fid50f4wx0o7us.jpg) nowhere in Audiobookshelf config folder (Docker setup).

**Comment by advplyr** [comment, marked as answer]
There was a bug that I fixed for the issue you put in. Updating the local image path as you were doing before will work in the next release.
Also, you can use a URL from the web and it will be downloaded to `/metadata/authors`.

What you are experiencing with the author image path not existing is in part due to the bug I just fixed but the reason you are still seeing an image is because it is cached in `/metadata/cache/images`

**Reply by advplyr** [reply]
As a temp workaround for now you can purge cache and Abs will be forced to cache a new author image using the path stored on the Author.

**Reply by Gajotres** [reply]
Thank you for the feedback.
